aad/azure-spring-boot-starter-active-directory/aad-resource-server/pom.xml

```diff
@@ -19,7 +19,7 @@
       <dependency>
         <groupId>com.azure</groupId>
         <artifactId>azure-core</artifactId>
-        <version>1.18.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+        <version>1.19.0</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
How about import bom instead of write these versions?

Reply:
I will do it this month